client/public/src/components/UserFunctions.js

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios'
+
+export const register = newUser => {
+    return axios
+        .post('users/register', {
```

Comment:
You could just use the `newUser` object:

```
return axios.post('users/register', newUser)
          .then(res => {
            console.log("Registered")
        }).catch(err => {
            console.log("error  " + err)
        })
```

Same goes for functions in this file.